aten/src/ATen/native/Normalization.cpp

```diff
@@ -155,6 +155,8 @@ std::tuple<Tensor,Tensor> batch_norm_cpu_update_stats_template(
     double momentum, double eps) {
 
   using accscalar_t = at::acc_type<scalar_t, false>;
+  TORCH_CHECK(input.dim() >= 1,
```

Comment:
Since `batch_norm_cpu_update_stats_template` is not a user-facing function this should be an internal assertion -- maybe the check could occur earlier?